CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## vNext
 
+- Development.
+- StackTrace factory : package are inApp by default 
```

Comment:
please align the changelog item + PR's title, a prefix would be nice like Fix: title

Reply:
✅